report/sql_tasks/tasks/report/create_from_scratch/01_functions/01_date_functions.sql

```diff
@@ -54,3 +54,6 @@ AS $$ SELECT
     END
 $$
 LANGUAGE SQL;
+
+-- We need to commit these functions, so they are available to Python scripts
+COMMIT;
```

Comment:
This is very iffy. Why aren't these available to the scripts anyway? Seems like something is wrong, they are all in the same process with the same connection.

I might have just got things in the wrong order?

Reply:
I tried without this and both create/refresh seemed to work.

Ideally we could remove this if not needed right now.